platform/macos/scripts/package.sh

```diff
@@ -43,7 +42,7 @@ if [[ -e ${PRODUCTS}/${BUILDTYPE}/${NAME}.framework.dSYM ]]; then
     cp -r ${PRODUCTS}/${BUILDTYPE}/${NAME}.framework.dSYM "${OUTPUT}"
 fi
 
-if [[ "${GCC_GENERATE_DEBUGGING_SYMBOLS}" == false ]]; then
+if [[ ${SYMBOLS} = NO ]]; then
     step "Stripping binaries…"
     strip -Sx "${OUTPUT}/${NAME}.framework/${NAME}"
```

Comment:
Would it be possible to do dSYM validation for macOS as well? I think it might’ve caught #5818 indirectly.